src/test/java/dk/kb/present/copyright/CopyrightAccessExtractorTest.java

```diff
@@ -65,6 +69,11 @@ void testSinglePerson() throws Exception {
         assertEquals(1831, mapper.getLastDeathYearForPerson());        
         assertEquals(true, mapper.isEjerMaerke());
         assertEquals(1831, mapper.getSkabelsesAar());
+                
+        assertFalse(mapper.isFotoAftale());        
+        assertTrue(mapper.isBilledeAftale());
+        assertEquals("Fri af ophavsret", mapper.getOphavsretTekst()); //TEMP!
+        System.out.println(mapper.getOphavsretTekst());
```

Comment:
Please remove or out-comment all `System.out.println` in tests as they are messy when doing a `mvn package`.

Reply:
indeed.

Reply:
indeed

Reply:
Double checking there is also a `System.out.println` in `CopyrightAccessDto2SolrFieldsMapper`.

And in general there's lots of prints in test classes, but as many asserts has been disabled there I guess that re-enabling those is a general issue.

Reply:
Fixed. 
Forbidden API should not allow System.out in classes under /src/main. Can this be configured?